tests/prague/eip7702_set_code_tx/test_set_code_txs_2.py

```diff
@@ -0,0 +1,403 @@
+"""
+A state test for [EIP-7702 SetCodeTX](https://eips.ethereum.org/EIPS/eip-7702).
+"""
+
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    AuthorizationTuple,
+    Block,
+    BlockchainTestFiller,
+    Environment,
+    StateTestFiller,
+    Storage,
+    Transaction,
+    compute_create_address,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from .spec import Spec, ref_spec_7702
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_7702.git_path
+REFERENCE_SPEC_VERSION = ref_spec_7702.version
+
+
+@pytest.mark.valid_from("Prague")
+def test_7702_pointer_contract_pointer_loop(state_test: StateTestFiller, pre: Alloc):
+    """
+    tx -> call -> pointer A -> contract A -> pointer B -> contract loop C
+
+    Call pointer that goes more level of depth to call a contract loop
+    Loop is created only if pointers are set with auth tuples
+    """
+    env = Environment()
+
+    storage: Storage = Storage()
+
+    sender = pre.fund_eoa()
+    pointer_a = pre.fund_eoa()
+    pointer_b = pre.fund_eoa()
+    contract_a = pre.deploy_contract(
+        code=Op.SSTORE(storage.store_next(1, "contract_a_worked"), 0x1)
+        + Op.CALL(1_000_000, pointer_b, 0, 0, 0, 0, 0)
+        + Op.STOP,
+    )
+
+    storage_loop = storage.store_next(0, "contract_loop_worked")
+    contract_loop = pre.deploy_contract(
+        code=Op.SSTORE(storage_loop, Op.ADD(1, Op.SLOAD(0)))
+        + Op.CALL(1_000_000, pointer_a, 0, 0, 0, 0, 0)
+        + Op.STOP,
+    )
+    tx = Transaction(
+        to=pointer_a,
+        gas_limit=100_000_000,
+        data=b"",
+        value=0,
+        sender=sender,
+        authorization_list=[
+            AuthorizationTuple(
+                address=contract_a,
+                nonce=0,
+                signer=pointer_a,
+            ),
+            AuthorizationTuple(
+                address=contract_loop,
+                nonce=0,
+                signer=pointer_b,
+            ),
+        ],
+    )
+
+    # TODO: Modify post-state allocations here.
+    post = {pointer_a: Account(storage=storage)}
+
+    state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+@pytest.mark.valid_from("Prague")
+def test_7702_pointer_to_pointer(state_test: StateTestFiller, pre: Alloc):
+    """
+    tx -> call -> pointer A -> pointer B
```

Comment:
```suggestion
    Tx -> call -> pointer A -> pointer B
```
Same comment as above, just to appease the linter.